README.md

```diff
@@ -178,6 +179,7 @@ Tip: If your build hangs, try to increase the memory available to Docker.
 [CaptureHDR-url]: source/Camera/Basic/CaptureHDR/CaptureHDR.cpp
 [CaptureHDRCompleteSettings-url]: source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp
 [CaptureHDRLoop-url]: source/Camera/Advanced/CaptureHDRLoop/CaptureHDRLoop.cpp
+[CaptureHDRPrintNormals-url]: source/Camera/Advanced/CaptureHDRPrintNormals/CaptureHDRPrintNormals.cpp
```

Comment:
I would call the sample CaptureHDRComputeNormals - the fact that you are printing them is not that important I think.

Reply:
both samples compute normals